FirmataParser.cpp

```diff
@@ -130,6 +130,9 @@ void FirmataParser::parse(uint8_t inputData)
           if (currentReportDigitalCallback)
             (*currentReportDigitalCallback)(currentReportDigitalCallbackContext, multiByteChannel, dataBuffer[0]);
           break;
+        case REPORT_VERSION:
+          if (currentReportVersionCallback)
+            (*currentReportVersionCallback)(currentReportVersionCallbackContext, dataBuffer[0], dataBuffer[1], (const char *)NULL);
```

Comment:
I'll add `QUERY_FIRMWARE_VERSION` and `QUERY_PROTOCOL_VERSION` for v3.0 since I can't really think of a good reason for a client library to send it's version to the board.